californium-core/src/main/java/org/eclipse/californium/core/network/stack/ReliabilityLayer.java

```diff
@@ -213,7 +213,7 @@ public void run() {
 	public void receiveRequest(final Exchange exchange, final Request request) {
 
 		if (request.isDuplicate()) {
-			if (exchange.getSendNanoTimestamp() > request.getReceiveNanoTimestamp()) {
+			if (exchange.getSendNanoTimestamp() > request.getNanoTimestamp()) {
 				// received before response was sent
 				int count = counter.incrementAndGet();
```

Comment:
I still not get why we count this :sweat_smile:

Reply:
Just to have a small statistic, if that pays off.